tools/embark-oss/README.md

```diff
@@ -0,0 +1,21 @@
+# Embark OSS tool
+
+This tool is used to validate that our open source projects adhere to our open source guidelines.
```

Comment:
This tool is used to validate that our open source projects adhere to open source guidelines.